python/ray/serve/tests/test_logging.py

```diff
@@ -644,5 +646,56 @@ def test_configure_component_logger_with_log_encoding_env_text(log_encoding):
         logger.handlers.clear()
 
 
+@pytest.mark.parametrize(
+    "ray_instance",
+    [
+        {"RAY_SERVE_DISABLE_STDOUT": "1"},
+    ],
+    indirect=True,
+)
+def test_logging_disable_stdout(serve_and_ray_shutdown, ray_instance):
+    """Test logging when RAY_SERVE_DISABLE_STDOUT is set.
+
+    When RAY_SERVE_DISABLE_STDOUT=1 is set, serve should redirect stdout and stderr to
+    serve logger.
+    """
+    # Clean up log directory.
+    logs_dir = Path("/tmp/fake_logs_dir")
+    if logs_dir.exists() and logs_dir.is_dir():
+        shutil.rmtree(logs_dir)
```

Comment:
what's this for? are you trying to use a temporary dir? if so, use `tempfile` (grep around tests for other instances)